### PR TITLE
feat: make register client consume registrar v3 api

### LINKED ISSUE #115: Update register_cli to match API changes in AtRegistrar
### Describe the bug

- The registrar has modified some API calls, update the methods to match the current behaviour of the registrar.
- Update the code in RegisterUtil class according to the new Registrar API responses
- Update exception catching and user-messages

### Expected behavior

Register should be able to activate atsigns and display accurate user messages

### Process Checklist

- [x] Modify RegisterUtil methods with new Registrar API call responses
- [x] Update docs/readme

### PR DESCRIPTION
**Pull request by srieteja**
closes https://github.com/atsign-foundation/at_java/issues/115
**- What I did**
- update incorrect outdated command to use at_java in readme
- changed the config.yaml values point to registrar v3 api
- rename all method/variable names that contained 'v3' -> 'superApi' for more clarity
- fixed log statement being printed twice which asks for user verification code
- minor reformatting of logs and user messages
- updated config_reader tests to match new config

**- How to verify it**
- register should be working without any issues

**- Description for the changelog**

feat: make register client consume registrar v3 api

### CODE REVIEW

File: at_client/src/main/java/org/atsign/client/cli/Register.java [hunk 1]
```diff
@@ -165,12 +167,12 @@ class GetFreeAtsign extends RegisterApiTask<RegisterApiResult<Map<String, String
 
     @Override
     public RegisterApiResult<Map<String, String>> run() {
-        System.out.println("Getting free atsign ...");
+        System.out.println("Fetching free atsign ...");
```
Comment (murali-shris) [inline at_client/src/main/java/org/atsign/client/cli/Register.java:170]: replace sysout with logger

File: at_client/src/main/java/org/atsign/client/cli/Register.java [hunk 2]
```diff
@@ -241,12 +243,12 @@ public RegisterApiResult<Map<String, String>> run() {
     }
 }
 
-class GetAtsignV3 extends RegisterApiTask<RegisterApiResult<Map<String, String>>> {
```
Comment (murali-shris) [inline at_client/src/main/java/org/atsign/client/cli/Register.java:244]: is anyone using this method GetAtsignV3? should we deprecate or remove?

File: at_client/src/main/java/org/atsign/client/cli/Register.java (hunk 1 above)
Comment (srieteja) [reply at_client/src/main/java/org/atsign/client/cli/Register.java:170]: at_java does not have a logger yet

Comment (gkc) [reply at_client/src/main/java/org/atsign/client/cli/Register.java:170]: Yeah we should have a separate ticket to introduce a proper logger. Not for this PR though

**Comment by gkc** [comment]
We should be using syserr for log messages instead of sysout though

**Review by murali-shris — APPROVED** [review]

File: at_client/src/main/java/org/atsign/client/cli/Register.java (hunk 2 above)
Comment (srieteja) [reply at_client/src/main/java/org/atsign/client/cli/Register.java:244]: GetAtsignV3 is the method that uses the superAPI key. But now all the API endpoints have been moved to registrarV3, so I have renamed to make it more clear.

**Review by gkc — APPROVED** [review]

**Pull request merged**